docs/mmteb/points/672.jsonl

```diff
@@ -0,0 +1,3 @@
+{"GitHub": "wissam-sib", "New dataset": 12}
```

Comment:
Not sure about this. Counted it as 4*2 + 4 bonus (first Italian Pair Classification). Let me know if I need to count it as 2 + 4 instead.

Reply:
It should be 2 for the dataset + 4 for the language bonus.